app/javascript/css/sharedTailwindClasses.js

```diff
@@ -0,0 +1,4 @@
+const buttonClasses =
```

Comment:
This probably could be wrapped in a css class rather than a variable?

Reply:
Yeah, unfortunately we can't wrap all of the Tailwind classes in a CSS class.  For example, here is one of the errors when trying to do so:

```
`@apply` cannot be used with `.hover` because it's actual definition includes a pseudo-selector
```

I suppose we could break out the hover classes (there are a few), and add them in to each button manually across all the components that use them, or create a `const` for them like I am doing.  But IMO if we have to go through those workarounds in order to bundle things in a CSS class, it makes more sense to just keep it all in one place in a `const`.

Normally this would be handled through a parent CSS or Sass class, but since we are using Tailwind classes, things are a little bit different.  So I'm cool with keeping things DRY for our developer sanity through a `const` in a shared file in the CSS directory.  

You know what I mean?

Reply:
I'll merge this PR to keep moving, and if we come up with a solution for this ☝️ let's for sure implement it in an upcoming PR!